packages/quantic/force-app/main/default/lwc/quanticResultLink/__tests__/quanticResultLink.test.js

```diff
@@ -0,0 +1,108 @@
+// @ts-nocheck
```

Comment:
Why do we need to completely disable all the type script checks?

Reply:
I see, well personally I prefer not to disable the whole ts checks because they can be useful sometimes but feel free to go with the option you judge best.